help/en/package/jmri/jmrit/beantable/LogixAddEdit.shtml

```diff
@@ -61,11 +61,13 @@
       names may be any useful text, provided the new user name is
       not used by another Conditional belonging to this Logix. It
       may be the same as the user name of another Logix's
-      Conditional. <b>When editing the user name (or any item in
+      Conditional.  <b>Note: The name must be unique among all
```

Comment:
The ```<b>``` and ```<i>``` HTML elements are stylistic and not semantic. If you want varying levels of emphasis, use ```<em>``` to indicate emphasis and ```<strong>``` to indicate strong emphasis.